core/server/models/post.js

```diff
@@ -676,6 +676,7 @@ Post = ghostBookshelf.Model.extend({
 
         // newsletter_id is read-only and should only be set using a query param when publishing/scheduling
         if (options.newsletter_id
+            && !this.get('newsletter_id')
```

Comment:
Thinking through whether this conditional is needed. If we keep it then it means the only way to change a newsletter for a scheduled post is to make the post a draft, then re-schedule. This makes sense within the existing publishing flow because we disable the recipient selection too but it might need re-visiting in the context of the publishing flow redesign.

No issue with it going in as-is but I'll ping design team to check.

Reply:
Given [this](https://github.com/TryGhost/Ghost/blob/7a55e1a60ad6a7a1d4d451a92a5f853857cd496b/core/server/models/post.js#L687-L688) I believe we can't change the `email_recipient_filter` while a post stays on `scheduled`. So this shouldn't be an issue, right?

Reply:
That's the same assumption I'm looking for clarity on with regards to the updated publishing flow.

Reply:
I'm reformulating to make sure I understand:
1. Right now we need to unpublished a scheduled post in order to change the recipients, which is enforced both in the Admin and in the API.
2. There is the option to change this flow so that we can edit the recipients of a post while it's scheduled, this needs Peter's input and if we go with it we'll need to change the server & admin implementation

Reply:
1. Correct.
2. Correct. We're re-working the publishing flow so all things like this are up for re-evaluation 🙂 Nothing needed in this PR so sorry for the noise, it was just seeing the conditional that triggered the thought 😅